tests/TestCase/Command/AllCommandTest.php

```diff
@@ -186,4 +186,46 @@ public function testExecuteWithPrefix()
         );
         $this->assertOutputContains('Bake All complete');
     }
+
+    /**
+     * Test docblock \@\uses generated for test methods
```

Comment:
Why are there \ in this?

Reply:
My IDE tried to annotate the word following the `@uses`. Codesniffer doesn't care apparently, so I can remove the slashes. Yes?

Reply:
Found other instances of `@uses` in code when not actually annotating. So I pushes a commit removing them.